anta/tests/snmp.py

```diff
@@ -237,3 +240,75 @@ def test(self) -> None:
             self.result.is_failure(f"Expected `{self.inputs.contact}` as the contact, but found `{contact}` instead.")
         else:
             self.result.is_success()
+
+
+class VerifySnmpLogging(AntaTest):
+    """Verifies whether the SNMP logging is enabled and SNMP manager(host) details in a specified VRF.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the SNMP logging is enabled and manager(host) details is in the specified VRF.
+    * Failure: The test will fail if the SNMP logging is disabled or the SNMP manager details are not correct.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.snmp:
+      - VerifySnmpLogging:
+          hosts:
+            - 192.168.1.100
+              vrf: default
+            - 192.168.1.101
+              vrf: MGMT
+    ```
+    """
+
+    name = "VerifySnmpLogging"
+    description = "Verifies whether the SNMP logging is enabled and SNMP manager(host) in a specified VRF"
+    categories: ClassVar[list[str]] = ["snmp"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show snmp", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifySnmpLogging test."""
+
+        hosts: list[SNMPHost]
+        """List of SNMP hosts."""
+
+        class SNMPHost(BaseModel):
+            """Model for a SNMP Host."""
+
+            hostname: IPv4Address
```

Comment:
if there is only option of ipv4 address then better to use host instead of hostname. If we can use and name then update input restriction